src/app/pages/profile-view/profile-view.component.html

```diff
@@ -0,0 +1,32 @@
+
+<div fxLayout="column" class="container">
+  <div class="pictures">
+    <div class="backround-picture" style="background-image: url('/assets/img/profile-pic-backround.jpg')">
+      <img class="profile-pic"  src="/assets/img/cedric.jpg"/>
+    </div>
+
+    <div fxLayout="row">
+      <div class="title" fxFlex="77%">
+        <h1>Cedric the beast</h1>
+        <h2>M2 Informatique DL</h2>
+      </div>
+  
+      <div class="social-media" fxFlex="auto">
+        <a href="https://www.facebook.com/alfyaounde/"> <img src="/assets/img/about/logoFacebook.png"
+          alt="Lien Facebook" width="42" height="42" hspace="10"> </a>
+      <a href="https://www.instagram.com/asso_alfy/"> <img src="/assets/img/about/logoInstagram.png"
+          alt="Lien Instagram" width="42" height="42" hspace="10"> </a>
+      <a href="https://twitter.com/associationalfy"> <img src="/assets/img/about/logoTwitter.png"
+          alt="Lien Twitter" width="42" height="42" hspace="10"> </a>
+      <a href="mailto:alfy@gmail.com"> <img src="/assets/img/about/logoEmail.png" alt="Lien Email" width="42"
+          height="42" hspace="10"> </a>
+      </div>
```

Comment:
Ici il y a pas mal de répétitions. Il est possible de créer, dans le profile-view-component.ts une liste des différents réseaux sociaux, par ex : 
public listSocialMedias = [
{href: "https://www.facebook.com/alfyaounde/", imgLastPath: "logoFacebook.png", altText: "Lien Facebook"},
{...}, etc
]

et du coup, dans le fichier profile-view-component.html : 
<div class="social-media" fxFlex="auto">
<a *ngFor="let socialMedia of listSocialMedias" href={{socialMedia.href}} <img src="/assets/img/about/{{socialMedia.imgLastPath}}" alt={{socialMedia.altText}} width="42" height="42" hspace="10"> </a>

Au moins il est plus facile de gérer les changements et l'évolution à partir du fichier .ts plutôt qu'html : rajouter un lien ne nécessite qu'un ajout d'un objet dans la liste.